catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,14 +13,36 @@
  **/
  -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
-        <!-- The tika input transformer programmatically registers itself as a service -->
+    <!-- The tika input transformer programmatically registers itself as a service -->
     <bean id="tikaTransformer" class="ddf.catalog.transformer.input.tika.TikaInputTransformer">
-		    <argument ref="blueprintBundleContext"/>
-		</bean>
+        <argument ref="blueprintBundleContext"/>
+        <argument ref="genericMetacardType"/>
+    </bean>
 
-    <reference-list id="contentExtractors" interface="ddf.catalog.content.operation.ContentMetadataExtractor"
+    <bean id="genericMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
```

Comment:
I don't like it either but I couldn't come up with anything better.  Basic?  Dataset?  Content?  I'd rather not call it tika since that is an implementation detail.